src/docfx/SubCommands/InitCommand.cs

```diff
@@ -384,18 +384,29 @@ private static bool ProcessOverrideQuestion(string message)
                         overrides = false;
                     }
                 });
-            overrideQuestion.Process(null, new QuestionContext());
+
+            if (overrides)
+            {
+                overrideQuestion.Process(null, new QuestionContext {NeedWarning = true});
```

Comment:
>overrideQuestion.Process(null, new QuestionContext {NeedWarning = true}); [](start = 16, length = 73)

how about remove if/else and write as overrideQuestion.Process(null, new QuestionContext {NeedWarning = overrides});